src/styles/polygons/polygons_fragment.glsl

```diff
@@ -35,9 +40,14 @@ void main (void) {
     vec4 color = v_color;
     vec3 normal = TANGRAM_NORMAL;
 
-    // Normal material texture (fragment lighting only)
-    #if defined(TANGRAM_LIGHTING_FRAGMENT) && defined(TANGRAM_MATERIAL_NORMAL_TEXTURE)
-        calculateNormal(normal);
```

Comment:
Ack, that is a mistake if we lost regular material normal maps! Let me look through history... but yes I think that allowing them to be compound makes sense.